src/profile/webid.test.ts

```diff
@@ -194,4 +195,34 @@ describe("getProfileAll", () => {
       getThingAll(MOCK_PROFILE)
     );
   });
+
+  it("deduplicates profile values", async () => {
+    const mockedFetch = jest.fn(fetch).mockResolvedValueOnce(
+      new Response(await triplesToTurtle(toRdfJsQuads(MOCK_PROFILE)), {
+        headers: {
+          "Content-Type": "text/turtle",
+        },
+        url: "https://some.profile",
+      } as ResponseInit)
+    );
+    // The profile document will have two triples <profile, foaf:primaryTopic, webid>...
+    const profileContent = buildThing({ url: "https://some.profile" })
+      .addIri(foaf.primaryTopic, MOCK_WEBID)
+      .build();
+    // and <webid, foaf:isPrimaryTopicOf, profile>.
+    const webidData = buildThing({ url: MOCK_WEBID })
+      .addIri(foaf.isPrimaryTopicOf, "https://some.profile")
+      .build();
+    const webIdProfile = [profileContent, webidData].reduce(
```

Comment:
I wouldn't recommend using `reduce` here, and instead just do the two (or three) discrete operations

Reply:
Done in ac286587. Do you think `reduce` is less readable than the alternative here ?